hydra-uber/src/main/resources/log4j2.conf

```diff
@@ -41,7 +44,7 @@ loggers: {
     { name: org.apache.xbean.spring, level: warn }
   ],
   root: { level: ${_out_min}, AppenderRef: [
-    { ref: STDERR, level: ${_err_min}},
-    { ref: STDOUT, level: ${_out_min}}
+    { ref: File,   level: ${_err_min}},
+    { ref: Async,  level: ${_out_min}}
```

Comment:
I think we should use a single appender, I think this current config will double-log error level messages.

Reply:
I agree that the current config would potentially double-log the error level messages. However, after examining the log file generated on a test cluster I found the [WARN] logs were logged just once.

I suspect the hocon property ${?logging.stderr} was set to "off" somewhere in the codebase, which means _err_min would be "off" instead of "warn". Thus, the err msg would just go through the "info" channel which is the async channel.

 I also think that the old config would potentially log the error level twice but because of the same reason (_err_min = ${?logging.stderr} = "off"), the log it generated actually had just one copy of the [WARN] msg. If you examine https://github.com/addthis/hydra/blob/master/hydra-uber/src/test/java/com/addthis/hydra/uber/LogTest.java you and run the test in Intellij you will find that the [WARN] is printed out twice and I believe that should be the case if ${?logging.stderr} is not set to "off"

Reply:
The decision I made was to keep change { ref: File,   level: ${_err_min}} into { ref: File,   level: ${_out_min}} and delete the async appenderRef